docs/src/requirements/kubernetes.md

```diff
@@ -1,9 +1,9 @@
 # Kubernetes requirements
 
-Zombienet should works with any k8s cluster (e.g [GKE](https://cloud.google.com/kubernetes-engine), [docker-desktop](https://docs.docker.com/desktop/kubernetes/), [kind](https://kind.sigs.k8s.io/)) **but** you need to have `kubectl` installed to interact with your cluster.
+Zombienet should work with any k8s cluster (e.g [GKE](https://cloud.google.com/kubernetes-engine), [docker-desktop](https://docs.docker.com/desktop/kubernetes/), [kind](https://kind.sigs.k8s.io/)) **but** you need to have `kubectl` installed to interact with your cluster.
 
 Also, you need _permission_ to create resources (e.g `namespaces`, `pods` and `cronJobs`) in the target cluster.
 
 ## Using `Zombienet` GKE cluster (internally).
 
-Zombienet project has it's own k8s cluster in GCP, to use it please ping _Javier_ (@javier:matrix.parity.io) in element to gain access and steps to use.
+The Zombienet project has it's own k8s cluster in GCP, to use it please ping [Javier](@javier:matrix.parity.io) in Element to get access and learn how to use it.
```

Comment:
The [Javier] link is wrong and never works. That is why I changed it in previous PR to non-link

Reply:
Ah ofcourse - I see. Will update, one sec. Sorry for the back and forth.